src/liquidity_pools/single_liquidity_pool_response.rs

```diff
@@ -0,0 +1,69 @@
+use derive_getters::Getters;
+use serde::{Deserialize, Serialize};
+
+use crate::models::Response;
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[serde(rename_all = "camelCase")]
+pub struct SingleLiquidityPoolResponse {
+    #[serde(rename = "_links")]
+    pub links: Links,
+    pub id: String,
+    #[serde(rename = "paging_token")]
+    pub paging_token: String,
+    #[serde(rename = "fee_bp")]
+    pub fee_bp: i64,
+    #[serde(rename = "type")]
+    pub type_field: String,
+    #[serde(rename = "total_trustlines")]
+    pub total_trustlines: String,
+    #[serde(rename = "total_shares")]
+    pub total_shares: String,
+    pub reserves: Vec<Reserve>,
+    #[serde(rename = "last_modified_ledger")]
+    pub last_modified_ledger: i64,
+    #[serde(rename = "last_modified_time")]
+    pub last_modified_time: String,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[serde(rename_all = "camelCase")]
+pub struct Links {
+    #[serde(rename = "self")]
+    pub self_field: Self_field,
+    pub transactions: Transactions,
+    pub operations: Operations,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[serde(rename_all = "camelCase")]
+pub struct Self_field {
```

Comment:
De struct `Self_field` mag ook een logischere naam krijgen, maar houdt er rekening mee dat `Self` ook een keyword is, in het Rust ecosysteem.